client/src/components/Banner/Banner.stories.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Banner from "./Banner";
+import InputTextField from "../InputTextField/InputTextField";
+import Button from "../Button/Button";
+
+const handleClick = e => {
+  console.log("Clicked");
+};
+
+storiesOf("Banner", module)
+  .add("Home Banner", () => <Banner
+    bannerType='var(--home-banner)'
+    bannerHeight='380px'
+  />)
```

Comment:
Can you remove `banner` from these prop names? It's redundant, because they're all props of the `Banner` component